go.mod

```diff
@@ -4,6 +4,8 @@ go 1.23.0
 
 toolchain go1.23.6
 
+replace github.com/longhorn/types => ../types
+
```

Comment:
@PhanLe1010 Why do we need the replacement?

Reply:
As mentioned in the PR description, we need to merge this dependent PR first https://github.com/longhorn/types/pull/45 then remove the `replace` directive